app/event/send-submission-batch-event.js

```diff
@@ -0,0 +1,29 @@
+const raiseEvent = require('./raise-event')
+const { v4: uuidv4 } = require('uuid')
+
+const sendSubmissionBatchEvent = async (batch, fileName) => {
+  const batchId = batch.batchId
+  for (const paymentRequest of batch.paymentRequests) {
+    const correlationId = paymentRequest?.correlationId ?? uuidv4()
+    const frn = paymentRequest?.frn ?? null
+    const paymentRequestNumber = paymentRequest?.paymentRequestNumber ?? null
+    const agreementNumber = paymentRequest?.agreementNumber ?? null
+    const event = {
+      id: correlationId,
+      name: 'payment-request-submission-batch-event',
+      type: 'submission',
+      message: 'Publish payment request batch',
```

Comment:
I would maybe say something to indicate that the payment request was batched for DAX as the current description is a bit ambiguous as to what has happened.